google/cloud/aiplatform/featurestore/feature.py

```diff
@@ -461,3 +461,141 @@ def search(
+        credentials: Optional[auth_credentials.Credentials] = None,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        sync: bool = True,
+    ) -> "Feature":
+        """Creates a Feature resource in an EntityType.
+
+        Example Usage:
+
+            my_feature = aiplatform.Feature.create(
+                feature_id='my_feature_id',
+                value_type='INT64',
+                entity_type_name='projects/123/locations/us-central1/featurestores/my_featurestore_id/\
+                entityTypes/my_entity_type_id'
+            )
+            or
+            my_feature = aiplatform.Feature.create(
+                feature_id='my_feature_id',
+                value_type='INT64',
+                entity_type_name='my_entity_type_id',
+                featurestore_id='my_featurestore_id',
+            )
+
+        Args:
+            feature_id (str):
+                Required. The ID to use for the Feature, which will become
+                the final component of the Feature's resource name, which is immutable.
+
+                This value may be up to 60 characters, and valid characters
+                are ``[a-z0-9_]``. The first character cannot be a number.
+
+                The value must be unique within an EntityType.
+            value_type (str):
+                Required. Immutable. Type of Feature value.
+                One of BOOL, BOOL_ARRAY, DOUBLE, DOUBLE_ARRAY, INT64, INT64_ARRAY, STRING, STRING_ARRAY, BYTES.
+            entity_type_name (str):
+                Required. A fully-qualified entityType resource name or an entity_type ID to create Feature in
+                Example: "projects/123/locations/us-central1/featurestores/my_featurestore_id/entityTypes/my_entity_type_id"
+                or "my_entity_type_id" when project and location are initialized or passed, with featurestore_id passed.
+            featurestore_id (str):
+                Optional. Featurestore to create Feature in.
+            description (str):
+                Optional. Description of the Feature.
+            labels (Dict[str, str]):
+                Optional. The labels with user-defined
+                metadata to organize your Features.
+                Label keys and values can be no longer than 64
+                characters (Unicode codepoints), can only
+                contain lowercase letters, numeric characters,
+                underscores and dashes. International characters
+                are allowed.
+                See https://goo.gl/xmQnxf for more information
+                on and examples of labels. No more than 64 user
+                labels can be associated with one Feature
+                (System labels are excluded)."
+                System reserved label keys are prefixed with
+                "aiplatform.googleapis.com/" and are immutable.
+            project (str):
+                Optional. Project to create Feature in. If not set, project
```

Comment:
Would it be ignored, or throw error if it is set and fully-qualified entity_type_name is used (may have different value for project)?

Same for location and featurestore id

Reply:
If `entity_type_name` is provided as fully-qualified resource name, the `project` and `location` args will be ignored.
Added description to indicate `project` and `location` are needed only when direct parent resource name arg is provided as a resource ID for both `create` methods in `EntityType` and `Feature`.